sormas-api/src/main/resources/strings.properties

```diff
@@ -725,15 +725,15 @@ infoCaseIncidenceIncompatible = No population data is available for communities
 infoNoPathogenTests = No pathogen tests have been created for this sample
 infoPickOrCreateCase = There are existing cases in the database that seem very similar to the one you are about to create. Please have a look at the list of existing cases and verify that the case you want to create is not a duplicate of one of them. If you find a case that is most likely the same as yours, please <b>click on it in the list</b> and confirm.
 infoPickOrCreateCaseNewCase = Newly added case information
-infoPickOrCreateImmunization = The system already contains an immunization for %s for this person that overlaps with the specified immunziation period. Please compare the existing immunization with the one you created and decide whether to discard the new immunization or use its information to update the existing one.
+infoPickOrCreateImmunization = The system already contains at least one immunization for COVID-19 for this person and means of immunization that overlaps with the specified immunziation period. Please compare the most recent existing immunization with the one you created and decide whether to discard the new immunization, use its information to update the existing one, or create it anyway.
```

Comment:
Remove the explicit disease name (the text in the issue was just an example) and replace it with a placeholder again (same in the string below)